modulefiles/module_gwsetup.hera.lua

```diff
@@ -14,5 +14,6 @@ load(pathJoin("python", python_ver))
 load("py-jinja2")
 load("py-pyyaml")
 load("py-numpy")
+load(pathJoin("git","2.40.0"))
```

Comment:
This should follow the paradigm of the others
```suggestion
local git_ver=os.getenv("git_ver") or "2.40.0"
load(pathJoin("git", git_ver))
```